#General considerations for designing primers to amplify template DNA:
- [] Length of 18-24 bases
- [] 40-60% GC content
- [] terminate 3' ends with G or C
- [] avoid stretches of 4 of the same nucleotide
- [] Melting temperature (Tm) of 55-65°C
- [] Primer pairs should have a Tm within 5°C of each other
- [] Primer pairs should not have complementary regions
- [] Any specific considerations for the polymerase you are using

##Using primers to add a restriction site:
- [] for 5' end preceed the restriction site with 5-8 extra nucleotides; for 3'end, follow with the extra nucleotides; this improves digestion efficiency.
- [] preferred cut sites can be viewed [here](https://www.neb.com/en-us/-/media/nebus/files/chart-image/cleavage_olignucleotides_old.pdf?rev=c2f94e1cdcd549c5bf8fdb59f7b63f67&hash=FB574C428654A26B052A85862F62B722)
- [] if your template is genomic DNA or plasmid, it is highly recommended to run your PCR product on an agarose gel, excise the desired band, and clean up to avoid inclusion of unwanted off-target DNA in downstream applications.
- [] the cleaned up PCR product can then be digested and cleaned up prior to cloning.
